gmailutils/gmail.go

```diff
@@ -253,27 +254,48 @@ func Subject(m *gmail.MessagePart) string {
 
 // NormalizeAndSplit normalizes subj format and split it to type/source.
 func NormalizeAndSplit(subj string) []string {
-	srcType, _ := splitOnDash(subj) // handles at least EN and FR locales
+	var srcType []string
+	srcType, _ = splitOnDash(subj) // handles at least EN and FR locales
 	if len(srcType) != 2 {
 		srcType = splitOnRuLocale(subj)
 	}
+
+	// nomalizes citations
+	if len(srcType) != 2 {
+		re := regexp.MustCompile(citations.En + `|` + citations.ja)
+		substr := re.FindAllStringSubmatch(subj, -1)
+		if substr != nil {
+			switch {
+			case substr[0][1] != "":
+				srcType = []string{substr[0][1], citations.En}
+			case substr[0][2] != "":
+				srcType = []string{substr[0][2], citations.En}
+			case substr[0][3] != "":
+				srcType = []string{substr[0][3], citations.En}
+			}
+		}
+	}
+
 	return srcType
 }
 
-type subjFormat struct{ ru, En string }
+type subjFormat struct{ ru, ja, En string }
 
 var (
 	articles = subjFormat{
-		"Новые статьи пользователя ", "new articles",
+		"Новые статьи пользователя ", "新しい論文", "new articles",
 	}
 	citations = subjFormat{
-		": новые ссылки", "new citations",
+		": новые ссылки", `^(?:(.+) さん|(自分))の論文からの引用: \d+ 件$`, `^\d+ new citations? to articles by (.+)$`,
```

Comment:
I wonder if the first one also needs to be updated with a regex for a number - I'll check that first.

Reply:
Thank you for the check! I don't have knowledge and testing environment for the language, so the update will be fine.